src/Spice86.Core/Emulator/Function/Dump/RecorderDataWriter.cs

```diff
@@ -36,19 +36,19 @@ public RecorderDataWriter(State state,
     /// <summary>
     /// Dumps all recorded data to their respective files.
     /// </summary>
-    public void DumpAll(ExecutionFlowRecorder executionFlowRecorder, FunctionHandler functionHandler) {
+    public void DumpAll(ExecutionFlowRecorder executionFlowRecorder, FunctionCatalogue functionCatalogue) {
         _loggerService.Verbose("Dumping all data to {DumpDirectory}", DumpDirectory);
```

Comment:
Could use a check if Verbose level is enabled first

Reply:
Changed to information and included it in an if